src/ResourceManager/AnalysisServices/Commands.AnalysisServices.ServiceManagement/Commands/AddAzureASAccount.cs

```diff
@@ -0,0 +1,122 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using System.IO;
+using System.Management.Automation;
+using System.Reflection;
+using System.Security;
+using Microsoft.Azure.Commands.AnalysisServices.ServiceManagement.Properties;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.AnalysisServices.ServiceManagement
+{
+    /// <summary>
+    /// Cmdlet to log into an Analysis Services environment
+    /// </summary>
+    [Cmdlet("Add", "AzureAsAccount", SupportsShouldProcess=true)]
```

Comment:
@shahabhijeet, this was the name our techinal architects and PMs came up with. I did ask them if we could use Add-AzureAnalysisServiceAccount, but they specifically wanted this. So I'll leave it as is.